tests/unit/services/test_logging.py

```diff
@@ -3,11 +3,14 @@
 from services.logging import timestamper
 
 
-def test_timestamper_with_debug(monkeypatch, freezer):
+def test_timestamper_with_debug(monkeypatch, time_machine):
     monkeypatch.setattr("services.logging.DEBUG", True)
 
+    dt = datetime.now()
+    time_machine.move_to(dt, tick=False)
```

Comment:
is there a reason you used `dt` here, rather than `now` as in other tests?

```suggestion
    now = datetime.now()
    time_machine.move_to(now, tick=False)
```

Reply:
Genuinely can't remember! Consistency ftw 👍